lib/movie_genres.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_project/styles.dart';
+
+import 'movie.dart';
+
+class MovieGenres extends StatelessWidget {
+  const MovieGenres({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+
+  static const double paddingBottomGenres = 15;
+  static const double genreFontSize = 15;
+  static const double borderRadiusGenreCard = 20;
+  static const double paddingGenreSpace = 12;
+
+  @override
+  Widget build(BuildContext context) {
+    const TextStyle genreText =
+        TextStyle(fontSize: genreFontSize, color: Colors.white54);
```

Comment:
trailing comma

Reply:
Done and also moved above the build widget